### DISCUSSION TITLE
How to get list of stashed changes

### DISCUSSION THREAD

**Discussion by guptagaurav0075** [gitpython-developers/GitPython#1165] [Q&A] [answered]
Documentation is throwing me off a little bit, can somebody please help in how we can perform the operation similar to the command: `git stash list `

Tried the below but it didn't work for list, but worked for drop and save.
```
from git import Repo
repo = Repo("Some Git Repo Path")
print(repo.git.stash("list"))
```

**Comment by guptagaurav0075** [comment]
The issue is resolved, Getting stash list as string Is there a list of strs that can be provided back?

**Comment by Byron** [comment, marked as answer]
There is no easier way to do that, as GitPython provides mostly low level commands.
I presume you will have to parse the output of the git command execution yourself.
As for the line splitting:

```
>>> "hi\nho".split("\n")
['hi', 'ho']
```
